src/main/java/pages/NewIssuePage.java

```diff
@@ -0,0 +1,61 @@
+package pages;
+
+import com.codeborne.selenide.Condition;
+
+import static com.codeborne.selenide.Selectors.byId;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.$;
+
+public class NewIssuePage {
+    private String inputProjectXpath = "//*[@id=\"project-field\"]";
+    private String inputIssueTypeXpath = "//*[@id=\"issuetype-field\"]";
+    private String inputSummaryXpath = "//*[@id=\"summary\"]";
+    private String inputDescriptionID = "description";
+    private String TextTabButtonXpath = "//*[@class=\"tabs-menu\"]/li[2]";
+    private String assignToMeButtonXpath = "//*[@id=\"assign-to-me-trigger\"]";
+    private String createButtonXpath = "//*[@id=\"create-issue-submit\"]";
+
+
+    public boolean atRequiredPage () {
+        $(byXpath("//*[@id=\"create-issue-dialog\"]/div[1]/h2")).should(Condition.visible);
+        return true;
+    }
+    public  void enterProjectName(String projectName){
+        $(byXpath(inputProjectXpath)).sendKeys(projectName);
+        $(byXpath(inputProjectXpath)).pressEnter();

```

Comment:
Try to rewrite the function in method:
$(byXpath(inputProjectXpath)).setValue(projectName).pressEnter();